changelog.md

```diff
@@ -1,11 +1,35 @@
 
+
+## 3.2.1 / 2020-11-04
+
+### New Features
+
+- Full implementation of spaces access types: open, on request and confidential (spaces available in Tracim < 3.2.1 are considering like type confidential)
+- Allow user to discover and join spaces
+- New plugins available:
+  - Adds every new user to all open spaces and to every newly created open space. You need to activate this plugin `tracim_backend_autoinvite` (documentation avialable [here](backend/official_plugins/README.md))
+  - Recursively adds new members of space to its parents with the default user role of each space. You need to activate this plugin `tracim_backend_parent_access` (documentation avialable [here](backend/official_plugins/README.md))
```

Comment:
```suggestion
  - Recursively adds new members of a space to its parents with the default user role of each space. You need to activate this plugin `tracim_backend_parent_access` (documentation avialable [here](backend/official_plugins/README.md))
```

Reply:
Can you check if this fix need to be added also in officla plugin README ? (if yes, I can change readme in this MR)

Reply:
This is already done in plugin readme. I had this fix in chnagelog.